src/app/pages/dashboard/widgets/cpu/widget-cpu-usage-recent/widget-cpu-usage-recent.component.ts

```diff
@@ -59,6 +59,9 @@ export class WidgetCpuUsageRecentComponent implements WidgetComponent {
   protected cpuStats = computed(() => {
     const initialStats = this.initialCpuStats();
     const cachedStats = this.cachedCpuStats();
+
+    if (!initialStats?.length || !cachedStats?.length) return null;
```

Comment:
I think this component was not affected by the issue since `initialCpuStats` initializes with an array with zeros and `cachedStats` is an empty array by default.